package.json

```diff
@@ -6,15 +6,19 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "@types/react": "^16.9.34",
     "antd": "^4.1.0",
     "bootstrap": "^4.4.1",
     "heroku": "^7.39.2",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-redux": "^7.2.0",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.4.1"
```

Comment:
My bad, this should be present in package.json. I thought create-react-app installs it as peer dependency, but it does not. Sorry, please, revert this line.